tests/component.js

```diff
@@ -413,3 +412,85 @@ describe( 'CKEditor Component', () => {
 		} );
 	}
 } );
+
+describe( 'comp on detach elem', () => {
```

Comment:
```suggestion
describe( 'component on detached element', () => {
```

I'd rather avoid using such shortcuts. It took me a while to get what "comp" means (at first I thought it was a shortcut of "compatibility").

Reply:
My bad :/